discovery_engine_core/providers/src/newscatcher_client.rs

```diff
@@ -0,0 +1,409 @@
+// Copyright 2022 Xayn AG
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, version 3.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+//! Client to get new documents.
+
+use std::{ops::Deref, time::Duration};
```

Comment:
Git picks this up as a new file, but this is just the previous `client.rs` renamed to `newscatcher_client.rs`